### ISSUE TITLE
env-shell inserts non-existant root path into LD_LIBRARY_PATH (Trac #1372)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1364]
When USE_ROOT is set to False env-shell.sh puts /USE_ROOT/IS/OFF/lib into the user's $LD_LIBRARY_PATH. This does no harm since no one is ever going to have that path, but introduces pointless clutter and wastes the dynamic linker's time. Maybe we could have it just skip adding the root library path when USE_ROOT is turned off. 

Migrated from https://code.icecube.wisc.edu/ticket/1372
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T00:03:39", 
    "description": "When USE_ROOT is set to False env-shell.sh puts /USE_ROOT/IS/OFF/lib into the user's $LD_LIBRARY_PATH. This does no harm since no one is ever going to have that path, but introduces pointless clutter and wastes the dynamic linker's time. Maybe we could have it just skip adding the root library path when USE_ROOT is turned off. ", 
    "reporter": "cweaver", 
    "cc": "david.schultz", 
    "resolution": "fixed", 
    "_ts": "1547251419341678", 
    "component": "cmake", 
    "summary": "env-shell inserts non-existant root path into LD_LIBRARY_PATH", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2015-09-30T19:03:27", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150930T19:04:55: 
* david.schultz changed milestone from "" to "offline V15-11-00"
* david.schultz changed priority from "trivial" to "minor"
* david.schultz changed cc from "" to "david.schultz"
* david.schultz changed status from "new" to "assigned"
* david.schultz changed owner from "" to "nega"

**Comment by icecube-trac** [comment]
Trac update at 20150930T19:21:42: 
* nega commented:

> In changeset 2323:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2323"
> clean paths when building w/o root
> 
> ack'd: cweaver
> closes #1364
> ```


* nega changed resolution from "" to "fixed"
* nega changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:10: 
* olivas changed milestone from "offline V15-12-00" to ""
* olivas commented:

> Milestone offline V15-12-00 deleted

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:03:39: nwhitehorn commented:

> In changeset 170137:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170137"
> clean paths when building w/o root
> 
> ack'd: cweaver
> closes #1364
> ```
